README.md

```diff
@@ -62,15 +59,15 @@ get all the submodules.
 git clone --recursive https://github.com/google/sling.git
 ```
 
-The parser trainer uses Tensorflow for training. SLING uses the Python 2.7
-distribution of Tensorflow, so this needs to be installed. The installed version
-of protocol buffers needs to match the version used by Tensorflow. Finally,
+The parser trainer uses Python v2.7 and PyTorch for training,
+so they need to be installed. Additionally,
 SLING uses [Bazel](https://bazel.build/) as the build system, so you need to
 install Bazel in order to build the SLING parser.
 
 ```shell
-sudo pip install -U protobuf==3.4.0
-sudo pip install https://storage.googleapis.com/tensorflow/linux/cpu/tensorflow-1.4.0-cp27-none-linux_x86_64.whl
+# Change to your favorite version as needed.
+sudo pip install http://download.pytorch.org/whl/cpu/torch-0.3.1-cp27-cp27mu-linux_x86_64.whl 
+sudo pip install torchvision
```

Comment:
Is torchvision strictly needed, and if so, what is it used for?

Reply:
You are right, not needed. This is a package for computer vision/image processing.